scripts/watch.cjs

```diff
@@ -40,11 +40,13 @@ const getWatcher = ({ name, configFile, writeBundle }) => {
   });
 };
 
+const EXTENSION_OPTION = '--extension';
```

Comment:
I would opt in for something meaning that we give a folder like --extension-folder or --extension-path